### PR TITLE
Add Openbox window manager

### PR DESCRIPTION
**Pull request by dsendkowski**

### CODE REVIEW

File: components/desktop/openbox/openbox.p5m [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+file path=usr/include/openbox/3.6/obt/xqueue.h
+link path=usr/lib/$(MACH64)/libobrender.so target=libobrender.so.32.0.0
+link path=usr/lib/$(MACH64)/libobrender.so.32 target=libobrender.so.32.0.0
+file path=usr/lib/$(MACH64)/libobrender.so.32.0.0
+link path=usr/lib/$(MACH64)/libobt.so target=libobt.so.2.0.2
+link path=usr/lib/$(MACH64)/libobt.so.2 target=libobt.so.2.0.2
+file path=usr/lib/$(MACH64)/libobt.so.2.0.2
+file path=usr/lib/$(MACH64)/pkgconfig/obrender-3.5.pc
+file path=usr/lib/$(MACH64)/pkgconfig/obt-3.5.pc
+file path=usr/libexec/openbox-autostart
```
Comment (alarcher) [inline components/desktop/openbox/openbox.p5m]: Executables in /usr/libexec need "mode=0555" added at the end of the line.

**Review by alarcher — CHANGES_REQUESTED** [review]
Tested on my machine.
You need to fix the 0555 mode but otherwise good.
Thank you.

File: components/desktop/openbox/openbox.p5m (hunk 1 above)
Comment (dsendkowski) [reply components/desktop/openbox/openbox.p5m]: Like this?:
```
file path=usr/libexec/openbox-autostart mode=0555
file path=usr/libexec/openbox-xdg-autostart mode=0555
```

Comment (alarcher) [reply components/desktop/openbox/openbox.p5m]: Yes!

Comment (alarcher) [reply components/desktop/openbox/openbox.p5m]: For reference, default transforms set the modes at https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/transforms/defaults#L105

Comment (dsendkowski) [reply components/desktop/openbox/openbox.p5m]: I added missing `python` and `python-xdg` dependencies, which emerged after specifying python version for `openbox-xdg-autostart`.

**Review by xen0l — APPROVED** [review]

**Review by alarcher — APPROVED** [review]

**Comment by alarcher** [comment]
@dsendkowski I just noticed that files were delivered in /usr/libexec. We do not use this folder: depending on the type of file either we put the binaries in /usr/bin or we put libraries in /usr/lib/$component/.

**Comment by pyhalov** [comment]
Usually we put them in /usr/lib. /usr/lib/$component was usually used for mate to avoid intersections with gnome.

**Comment by alarcher** [comment]
In this particular case libexecdir may be set to /usr/bin to be consistent with other components;
```
file path=usr/libexec/openbox-autostart
file path=usr/libexec/openbox-xdg-autostart
```

**Comment by dsendkowski** [comment]
OK, but first I'll check whether moving these files requires any additional changes.

**Comment by alarcher** [comment]
Sure. This is not a priority, no rush. It would just be nice to fix it at some point.

**Pull request merged**